tests/spec/modules/options/option-item-int.spec.js

```diff
@@ -0,0 +1,36 @@
+
+import { getOptionItemWith } from '.';
+
+describe("spec OptionItem:", () => {
+
+  describe("which has boolean value: ", () => {
+
+    beforeEach(() => {
+      this.optionItem = getOptionItemWith('number');
+      document.body.innerHTML = __html__['options.html'];
+    });
+
+    it("ele.value is expected when setValueOnElement is called with a number value", () => {
+      this.optionItem.value = 1000;
+      this.optionItem.setValueOnElement();
+      expect(this.optionItem.getElement().value)
+        .toBe(this.optionItem.value.toString());
+    });
+
+    it("return number value when _getTargetValue is called", done => {
```

Comment:
`return a number when _getTargetValue is called with a object which include 'value' property has a passable value as int.`